styles/globals.css

```diff
@@ -5,3 +5,8 @@
 @tailwind utilities;
 
 @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@100;300;400;500;600;700&display=swap');
+@import url('https://unpkg.com/leaflet@1.7.1/dist/leaflet.css');
+
+.leaflet-container {
```

Comment:
Just for your information, one of the creators of Tailwind twittered a couple of weeks ago that the use of @apply is not recommended. Most of the time you can circumvent it, e.g. by extracting things to components.